### ISSUE TITLE
--local runs are broken

### ISSUE THREAD

**Issue by alexander-fenster** [googleapis/artman#649]
After #637, the `--local` runs don't work since the logic of `install_protoc.sh` does not work outside of the carefully crafted Docker image. We need to use `install_protoc.sh` in Docker only, and if it does not exist, not fail but revert to using the `protoc` installed in the system.

**Comment by alexander-fenster** [comment]
@hzyi-google so since @andreamlin is out, do you think you'll have bandwidth to fix it, or should I just go ahead and do it? We do have folks using `--local` so they will complain soon I guess.

Also, I don't like the idea that the `.sh` script is being parsed in Python to extract the version numbers; I'd better refactor the version numbers away from the shell script but it's out of scope of this issue.

**Comment by hzyi-google** [comment]
Hey Alex,

If you have time please go ahead and fix it. I am on vacation for the rest
of the week and the entire week next week.

Thanks,
Hanzhen

**Comment by vam-google** [comment]
@alexander-fenster I'm wondering why that change was introduced. It looks like older version used by java was the case. Why didn't we just bumped the java protoc and grpc version, instead of doing that change. I would strongly consider reverting the change, as it complicates the system even more
